bigchaindb/pipelines/block.py

```diff
@@ -1,6 +1,6 @@
 """This module takes care of all the logic related to block creation.
```

Comment:
It was renamed to `BlockPipeline` as I introduced another class called `Block` that is the actual block.

Since we're handling `Block`s in `BlockPipeline`, I thought a separation of the two would be healthy for readability.